index.html

```diff
@@ -1,47 +1,83 @@
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/angular_icon.svg" />
+      <span class="tech-title">AngularJS</span>
+      <span class="release">Release: 2010</span>
+      <p class="text">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs
+        with an MVC
+        architecture.</p>
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/vue_icon.svg" />
+      <span class="tech-title">VueJS</span>
+      <span class="release">Release: 2014</span>
+      <p class="text">Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript framework for building user
+        interfaces.</p>
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/elm_icon.svg" />
+      <span class="tech-title">Elm</span>
+      <span class="release">Release: 2016</span>
+      <p class="text">Elm is a domain-specific programming language for declaratively creating web browser-based
+        graphical user
+        interfaces.</p>
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/polymer_icon.svg" />
+      <span class="tech-title">Polymer</span>
+      <span class="release">Release: 2015</span>
+      <p class="text">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+    </article>
+  </div>
+  <h1 class="title-tech">Back-End Tools</h1>
+  <div class="content">
+    <article class="tech">
+      <img class="tech-logo" src="assets/rails_icon.png" />
+      <span class="tech-title">Ruby on Rails</span>
+      <span class="release">Release: 2005</span>
+      <p>Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC
+        framework, providing default structures for a database, a web service, and web pages.</p>
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/express_icon.svg" />
+      <span class="tech-title">NodeJS</span>
+      <span class="release">Release: 2009</span>
+      <p>Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking
+        I/O model taht makes it lightweight and efficient.</p>
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/aspnet_icon.png" />
+      <span class="tech-title">ASP.NET Core</span>
+      <span class="release">Release: 2016</span>
+      <p>ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET
+        Framework, on Windows, and the cross-platform .NET Core.</p>
+    </article>
+    <article class="tech">
+      <img class="tech-logo" src="assets/django_icon.svg" />
+      <span class="tech-title">Django</span>
+      <span class="release">Release: 2005</span>
+      <p>Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the
```

Comment:
Same, `<p>` should be in the first like, then another line/s with string and then `</p>` in a new line